Doc/library/pathlib.rst

```diff
@@ -381,6 +381,12 @@ Pure paths provide the following methods and properties:
       >>> PureWindowsPath('//some/share').name
       ''
 
+   Trailing dots are not considered as filenames::
+
+      >>> PurePosixPath('..').name
+      ''
+
+
```

Comment:
I think there is one more newline here. What do you think about it?

Reply:
Yep, I missed it.... 
And by reading it again i chose an other phrasing that seems more appropriate....
Thank you,